cloudbuild-tag.yaml

```diff
@@ -8,23 +8,51 @@ steps:
   args:
   - -exc
   - |
-    docker pull gcr.io/marketplace-k8s-app-tools/k8s/dev:sha_$COMMIT_SHA
-    docker tag gcr.io/marketplace-k8s-app-tools/k8s/dev:sha_$COMMIT_SHA gcr.io/$PROJECT_ID/k8s/dev:$TAG_NAME
+    docker pull gcr.io/$_STAGING_PROJECT_ID/k8s/dev:sha_$COMMIT_SHA
```

Comment:
Is there a place we can see all of the triggers that will need to be retrofitted?

Maybe that design doc should have a section for it?

Reply:
This file affects only one trigger, in `cloud-marketplace-tools` prod project, which has already been retrofitted.